lib/collection/src/shards/queue_proxy_shard.rs

```diff
@@ -655,7 +655,10 @@ async fn transfer_operations_batch(
 ) -> CollectionResult<()> {
     // TODO: naive transfer approach, transfer batch of points instead
     for (_idx, operation) in batch {
-        let mut operation = operation.clone();
+        // TODO: Assign current peer to operation debug metadata!
+        let mut operation = operation
+            .clone()
+            .reason(crate::operations::Reason::WalDeltaTransfer);
```

Comment:
Any reason we're not doing this right now?

Reply:
Yeah. It's not propagated anywhere near `QueueProxy` ATM 😂